setup.py

```diff
@@ -46,11 +98,15 @@ def get_version():
     url='https://github.com/scikit-hep/probfit',
     package_dir={'probfit': 'probfit'},
     packages=['probfit'],
-    ext_modules=extensions,
+    ext_modules=get_extensions(),
+    setup_requires=[
+        'numpy',
+    ],
```

Comment:
No, this is not the correct way to do this, `setup_requires` is deprecated and shouldn't be _added_. Use pyproject.toml. This will also build with the wrong version of NumPy so the wheels will not be importable on an older version. And it doesn't install Cython, either....